addons/website_form/views/website_form_templates.xml

```diff
@@ -1,140 +1,184 @@
+                                                            <span class="s_website_form_label_content">Email To</span>
+                                                        </label>
+                                                        <div class="col-sm">
+                                                            <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to" />
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                                <div class="form-group col-12 s_website_form_submit" data-name="Submit Button">
+                                                    <div style="width: 200px;" class="s_website_form_label" />
+                                                    <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send">Submit</a>
+                                                    <span id="s_website_form_result"></span>
+                                                </div>
+                                            </div>
+                                        </form>
                                     </div>
-                                </div>
-                                <div class="form-group col-12 s_website_form_submit" data-name="Submit Button">
-                                    <div style="width: 200px;" class="s_website_form_label"/>
-                                    <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send">Submit</a>
-                                    <span id="s_website_form_result"></span>
-                                </div>
+                                </section>
                             </div>
-                        </form>
+                            <div class="col-lg-4 mt-4 mt-lg-0">
+                                <address itemscope="itemscope" itemtype="http://schema.org/Organization">
+                                    <p class="p-0 m-0">My Company</p>
+                                    <ul class="list-unstyled mb-0 pl-2">
+                                        <li><i class="fa fa-map-marker fa-fw mr-2" /><span class="o_force_ltr">Chaussée de Namur 40</span></li>
+                                        <li><i class="fa fa-phone fa-fw mr-2" /><span class="o_force_ltr">+ 32 81 81 37 00</span></li>
+                                        <li><i class="fa fa-1x fa-fw fa-envelope mr-2" /><span>info@yourcompany.example.com</span></li>
+                                    </ul>
+                                </address>
+                            </div>
+                        </div>
                     </div>
                 </section>
+                <div class="oe_structure mt-2" />
```

Comment:
See you should not have an oe_structure inside an oe_structure, what's the point? :wink: You can already drop snippet in the parent one.